packages/block-library/src/cover/style.scss

```diff
@@ -10,6 +10,7 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
+	overflow: hidden;
```

Comment:
Why is this needed for both image and video?

Reply:
This is only needed for the videos. But it does not affect the images or changes anything if also applied to images. We don't have a way in CSS to target a class if it contains a specific child e.g: a video, so I applied to both images and video.